### DISCUSSION TITLE
square images

### DISCUSSION THREAD

**Discussion by yahavshalem** [t1m0thyj/WinDynamicDesktop#310] [Ideas]
I frequently switch between portrait and landscape mode and would like my background to fit it. I use the big Sur landscape theme, which I know has square source photos, I would like it if the theme used those, in the highest definition possible.

**Comment by t1m0thyj** [comment]
You can download square versions of the macOS dynamic wallpapers from the WDD-mac-themes repo: https://github.com/t1m0thyj/WDD-mac-themes/releases/tag/big-sur

**Reply by yahavshalem** [reply]
I did that, and it seems like the quality has been downgraded. I use a 1080p monitor, but it still seems as if the pictures are a way lower resolution. Can you make one with the square images in the highest quality possible?

**Reply by t1m0thyj** [reply]
Does the image quality look bad if you open one of the JPEGs from the square theme in a photo viewer app? Or does it only appear bad when the image is used as a wallpaper? If the latter, you may want to [disable JPEG compression](https://github.com/t1m0thyj/WDD-scripts/tree/master/registry#disableenable-jpeg-compression).
